src/NHibernate.JetDriver.Tests/JetTestBase.cs

```diff
@@ -36,7 +36,7 @@ protected JetTestBase(bool autoCreateTables)
                 .SetProperty(Environment.ConnectionProvider, typeof(DriverConnectionProvider).FullName)
                 .SetProperty(Environment.ShowSql, "true")
                 .SetDefaultNamespace("NHibernate.JetDriver.Tests.Entities")
-                .SetProperty(Environment.ConnectionString, string.Format("Provider=Microsoft.ACE.OLEDB.12.0;Data Source={0};", DataFile));
+                .SetProperty(Environment.ConnectionString, string.Format("Provider = Microsoft.Jet.OLEDB.4.0;Data Source={0};", DataFile));

```

Comment:
This is not needed. This addresses the test run failures on x64 environments, but all that's needed to do is to change the Test project build from Any CPU to x86.